packages/admin-next/dashboard/src/v2-routes/api-key-management/api-key-management-detail/api-key-management-detail.tsx

```diff
@@ -19,14 +21,16 @@ export const ApiKeyManagementDetail = () => {
     return <div>Loading...</div>
   }
 
+  const isPublishable = api_key?.type === ApiKeyType.PUBLISHABLE
+
   if (isError) {
     throw error
   }
 
   return (
     <div className="flex flex-col gap-y-2">
       <ApiKeyGeneralSection apiKey={api_key} />
-      <ApiKeySalesChannelSection apiKey={api_key} />
+      {isPublishable && <ApiKeySalesChannelSection apiKey={api_key} />}
```

Comment:
very NIT, but it might be good to rename `ApiKeySalesChannelSection` to `PublishableApiKeySection`, since we can add more than sales channel to publishable keys